### DISCUSSION TITLE
有办法避免自动添加的3条规则么

### DISCUSSION THREAD

**Discussion by dqbpqdpb** [vernesong/OpenClash#3820] [Q&A]
![image_2024-03-20_14-37-46](https://github.com/vernesong/OpenClash/assets/8680332/5808b221-db26-4ec6-92b9-970ec9725284)
- DST-PORT,7895,REJECT
- DST-PORT,7892,REJECT
- IP-CIDR,198.18.0.1/16,REJECT,no-resolve
openclash似乎会自动添加这3条规则，第三条我猜是给FAKEIP用的吧？因为我用redir的，想说有办法去掉第三条么？

**Comment by blzadx** [comment]
抱歉，恐怕你要失望了，clash内核很久以前就由于种种原因放弃支持redir增强模式了，现在的redir是把fake-ip-filter里的域名列表填*，也就是不返回fake-ip，所有域名都返回真实ip，跟原先的redir的效果是一样的。
